docs/plugins.rst

```diff
@@ -307,6 +307,10 @@ any of the available LDAP servers, they will fall back to normal core
 authentication. Documentation of the LDAP standard in general can be found
 `here <https://en.wikipedia.org/wiki/Lightweight_Directory_Access_Protocol>`_.
 
+Installation of this plugin requires some system libraries:
+* On Ubnutu 18.04, install the ``libldap2-dev`` and ``libsasl2-dev`` APT packages.
+* On RHEL (CentOS) 7, install the ``openldap-devel`` and ``cyrus-sasl-devel`` RPM packages.
```

Comment:
Why provide RHEL advice here? Everything else is Ubuntu. I think it's better to not have any RHEL leakage.

Reply:
I don't feel strongly. Perhaps this knowledge was hard-won, and we want to preserve it? Since this isn't the overall installation tutorial, I'd be more ok with including information about other platforms.

@zachmullen is both the author of this plugin and perhaps one of the most likely to need to install on a RHEL system. I'm happy to defer to him.

Reply:
I don't have strong feelings either. This plugin is unusual in that it requires native (non-python) shared libraries to be available, so to me it's OK that it has these instructions. But if we don't want to enumerate platforms, we can just remove these and say "install the following shared libraries on your system" and then enumerate them. (I think it's just `libldap` and `libsasl` but I can't remember for sure.) At which point it'll be up to each user to google it for themselves.

Reply:
I withdraw my request, reasoning is: it's possibly better to preserve this info than it is to preserve purity.